code/_onclick/item_attack.dm

```diff
@@ -26,12 +26,13 @@
 
 /mob/living/proc/attacked_by(obj/item/I, mob/living/user, def_zone)
 	var/dmgcheck = apply_damage(I.force, I.damtype, def_zone)
-	if(!dmgcheck)
+	var/fakedmgcheck = apply_damage (I.fakeforce, I.fakedamtype, def_zone)
+	if(!dmgcheck && !fakedmgcheck)
```

Comment:
Thrown items will still deal 100% brute since this doesn't touch throwforce at all.